src/components/autozoomControl.ts

```diff
@@ -225,6 +234,22 @@ export default class AutozoomControl implements ICnnControl {
     }
   }
 
+  async _setMode(mode: AutozoomModes) {
```

Comment:
One drawback with the current implementation is that this method is private and only called on changes to options. In theory this could lead to an error state where `AutozoomController._options` does not match with the camera and it being hard to correct.